.github/workflows/build.yml

```diff
@@ -191,7 +191,7 @@ jobs:
       - run: python3 -m pip install build && python3 -m build
       - uses: actions/upload-artifact@v4
         with:
-          name: artifact-python-base
+          name: nonrelease-artifact-python-base
```

Comment:
since we're filtering on `artifact-*` this should probably just be called `python-base` (the `artifact-` part is redundant already!)